Pds4File/pds4file.py

```diff
@@ -2968,37 +2968,37 @@ def volset_pdsfile(self, category=None, rank=None):
     ### typing them wrong.
 
     @property
-    def is_volume_dir(self):
-        """True if this is the root level directory of a volume."""
-        #return (self.volname_ and not self.interior)
-        return (self.volname_ and not self.interior or False) # MJTM: 'or False' account for bundle sets
+    def is_bundle_dir(self):
+        """True if this is the root level directory of a bundle."""
+        return (self.bundlename_ and not self.interior) # Note that a bundle set will return an empty string '' rather than False
+        #return (self.volname_ and not self.interior or False) # MJTM: 'or False' account for bundle sets
 
     @property
-    def is_volume_file(self):
-        """True if this is a volume-level checksum or archive file."""
-        #return (self.volname and not self.volname_)
-        return (self.volname and not self.volname_ or False) # MJTM: 'or False' account for bundle sets
+    def is_bundle_file(self):
+        """True if this is a bundle-level checksum or archive file."""
+        return (self.bundlename and not self.bundlename_) # Note that a bundle set will return an empty string '' rather than False
+        #return (self.volname and not self.volname_ or False) # MJTM: 'or False' account for bundle sets
 
     @property
-    def is_volume(self):
+    def is_bundle(self):
         """True if this is a volume-level file, be it a directory or a
```

Comment:
Update the function comments for these functions:
```
is_bundle
is_bundleset_dir
is_bundleset_file
```

Reply:
Thanks, I hadn't been using the preload function previously so there weren't any errors reported:

```
In [1]: import pds4file

In [2]: a = pds4file.PdsFile.from_abspath('/Users/mm9684/DropboxSETI/Dropbox (SETI Institute)/Pds4FileTest/pds4-holdings/bundles/uranus_occs_ea
   ...: rthbased/uranus_occ_u0_kao_91cm/data/atmosphere/u0_kao_91cm_734nm_counts-v-time_atmos_ingress.xml')

In [3]: a.exists
Out[3]: True

In [4]: a.bundlename
Out[4]: 'uranus_occ_u0_kao_91cm'

In [5]: a.bundleset
Out[5]: 'uranus_occs_earthbased'

In [6]: a.is_bundleset_file
Out[6]: False

In [7]: a.is_bundle_file
Out[7]: False
```
But when I do use preload() I reproduce the same error, which I will fix by updating all instances of `self.volname*` and `self.volset*` to their "bundle" equivalents